src/SFA.DAS.SharedOuterApi/Infrastructure/ApiClient.cs

```diff
@@ -24,20 +24,18 @@ public class ApiClient<T> : IApiClient<T> where T : IInnerApiConfiguration
             IWebHostEnvironment hostingEnvironment,
             IAzureClientCredentialHelper azureClientCredentialHelper)
         {
-            _httpClient = httpClientFactory.CreateClient();
+            _httpClientFactory = httpClientFactory;
```

Comment:
```
_httpClient = httpClientFactory.CreateClient(typeof(T).Name);
```
This links to the above, having them all named causes no issues and will also mean that all of the other changes aren't necessary. Its more likely that the throttling would need to be done across every client